✨ add github actions for some ci checks

noahp opened the pull request:
Check on python 3.6 + 3.7 + 3.8:
- pylint
- black
- isort

Add necessary infrastructure and README.md info about how to use it
locally.

Fixes #5.

noahp commented:
See output here:
https://github.com/noahp/polly-merge/runs/653385595?check_suite_focus=true#step:3:312

mvertescher commented on README.md:
````diff
@@ -55,9 +65,23 @@ POLLY_MERGE_LOG_FILE=<your log file location>
+
+You can run whatever tests CI runs with these commands (requires Docker):
+
+```bash
````
Maybe we should add a dedicated script for this (per usual)... that might help reduce the copy paste here and in the Github CI. It would also make it easier to test locally :thinking: ...

noahp replied:
SGTM, making that change.

mvertescher commented on pyproject.toml:
```diff
@@ -0,0 +1,29 @@
+]
+build-backend = 'setuptools.build_meta'
+
+[tool.pylint.'MESSAGES CONTROL']
```
:ok_hand:

mvertescher commented on README.md:
```diff
@@ -1,10 +1,19 @@
+pull requests that have a comment/description containing `@polly <COMMAND>`
+(configurable).
+
+_NOTE: this is for use with the Bitbucket Server 1.0 REST API, it won't work
```
:smile: this is helpful!

mvertescher reviewed (approved):
Nice! Thanks for fixing up #5 quickly @noahp!

noahp committed:
✨ add github actions for some ci checks

Check on python 3.6 + 3.7 + 3.8:
- pylint
- black
- isort

Add necessary infrastructure and README.md info about how to use it
locally.

Fixes #5.
